projects/storefrontlib/src/translations/en/checkout-progress.en.ts

```diff
@@ -1,13 +1,8 @@
 export const checkoutProgress = {
```

Comment:
I would just add checkoutProgress main key to existing checkout chunk instead of creating new one.